docs/event-generator_run.md

```diff
@@ -31,20 +31,21 @@ event-generator run [regexp] [flags]
   -h, --help                           help for run
       --insecure-skip-tls-verify       If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
       --kubeconfig string              Path to the kubeconfig file to use for CLI requests.
+      --loop                           Run in a loop
       --match-server-version           Require server version to match client version
   -n, --namespace string               If present, the namespace scope for this CLI request (default "default")
       --request-timeout string         The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests. (default "0")
   -s, --server string                  The address and port of the Kubernetes API server
-      --sleep duration                 time to sleep prior to trigger an action
+      --sleep duration                 Time to sleep prior to trigger an action
```

Comment:
I would add an example e.g. (1s) as it's not obvious you need to provide units to the utility

Reply:
It also might be useful to put a default of 1s when we get --loop as otherwise it will loop as fast as it can

Reply:
You're right :+1: 

Maybe the simplest solution would be adding 1s as default:
```
--sleep duration                 Time to sleep prior to triggering an action (default "1s")
```